src/assets/logos/GSW.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const GSW = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      width="100%"
+      height="100%"
```

Comment:
👍 , Looks like it fixed the GSW bug, I will still use that other issue to fix all logos better in the circles. Good stuff

Reply:
Hm. I didn't fix the GSW bug on my browser, at least not in the recent game section. But whatev, good enough for now.

Reply:
Hmm, well, it's an anomaly, I'll take a look